x-pack/plugins/aiops/public/components/change_point_detection/use_change_point_agg_request.ts

```diff
@@ -165,18 +168,34 @@ export function useChangePointResults(
           query
         );
 
+        const metricFieldDV = metricFieldOptions.find(
+          (option) => option.name === fieldConfig.metricField
+        );
+        const splitFieldDV = splitFieldsOptions.find(
+          (option) => option.name === fieldConfig.splitField
+        );
+
+        requestPayload.body!.runtime_mappings = {
```

Comment:
Suggestion to pass runtimeMappings as a param to `getChangePointDetectionRequestBody()` to get the complete request. I don't think there's any need to call requestPayload and then append it to the payload.

Reply:
updated https://github.com/elastic/kibana/pull/168249/commits/0a2963c453399cb2ace90b80a0cddb30d78fe87c